.travis.yml

```diff
@@ -7,6 +7,10 @@ php:
   - 7
   - hhvm
 
+env:
+  - DOCTRINE_ORM="~2.4.0"
+  - DOCTRINE_ORM="~2.5.0"
```

Comment:
I'd rather bump the dependency to the ORM to 2.5 than introducing this sort of maintenance complexity.